src/analysis/analyzer.ts

```diff
@@ -89,7 +89,7 @@ export class Analyzer extends AnalyzerGen implements vs.Disposable {
 			// This will include things like Observatory output and some analyzer logging code.
 			message = message.trim();
 			if (!message.startsWith('--- ') && !message.startsWith('+++ ')) {
-				logError({ message: `Unable to parse message (${e}): ${message}` });
+				console.error(`Unable to parse message (${e}): ${message}`);
```

Comment:
Ah, ok, I haven't used that. Is there a better way to separate AS responses we care about and that sort of output? Feels a bit hacky to be attempting `JSON.parse` on every messages and assuming those that work are for us and those that don't are not?